src/plugins/discover/public/embeddable/components/search_embeddable_grid_component.tsx

```diff
@@ -81,15 +79,28 @@ export function SearchEmbeddableGridComponent({
     api.savedSearch$,
     api.savedObjectId,
     api.fetchWarnings$,
-    api.query$,
-    api.filters$,
     api.fetchContext$,
     stateManager.rows,
     stateManager.totalHitCount,
     stateManager.columnsMeta,
     stateManager.grid
   );
 
+  // `api.query$` and `api.filters$` are the initial values from the saved search SO (as of now)
```

Comment:
If `api.query$` and `api.filters$` already contain the saved search values, why are we switching to `searchSource` below?

Reply:
It seemed less verbose to me. Okay, I am going to keep the variables but reassign them. to api values https://github.com/elastic/kibana/pull/197190/commits/4615e64c4f7c4fdbf7fb80f879faa3ba66d7de43